stacker/config/__init__.py

```diff
@@ -289,6 +284,29 @@ class Stack(Model):
 
     profile = StringType(serialize_when_none=False)
 
+    external = BooleanType(default=False)
+
+
+class ExternalStack(BaseStack):
+    fqn = StringType(serialize_when_none=False)
+
+    @classmethod
+    def _claim_polymorphic(cls, value):
+        if value.get("external", False) is True:
+            return True
+
+    def __init__(self, *args, **kwargs):
+        super(ExternalStack, self).__init__(*args, **kwargs)
+        self.external = True
+
+    def validate_fqn(self, data, value):
+        if (not value and not data["stack_name"]) or \
```

Comment:
I just noticed this is wrong, since we can infer the CF name from the stack name. Will fix!